src/github/tests/test_utils.rs

```diff
@@ -176,6 +250,94 @@ impl TeamDataBuilder {
     }
 }
 
+#[derive(Clone, Builder)]
+#[builder(pattern = "owned")]
+pub struct RepoData {
+    name: String,
+    #[builder(default)]
+    pub description: String,
+    #[builder(default)]
+    pub homepage: Option<String>,
+    #[builder(default)]
+    bots: Vec<Bot>,
+    #[builder(default)]
+    pub teams: Vec<v1::RepoTeam>,
+    #[builder(default)]
+    pub members: Vec<v1::RepoMember>,
+    #[builder(default)]
+    pub archived: bool,
+    #[builder(default)]
+    pub allow_auto_merge: bool,
+}
+
+impl RepoData {
+    pub fn new(name: &str) -> RepoDataBuilder {
+        RepoDataBuilder::default().name(name.to_string())
+    }
+
+    pub fn add_member(&mut self, name: &str, permission: RepoPermission) {
+        self.members.push(v1::RepoMember {
+            name: name.to_string(),
+            permission,
+        });
```

Comment:
this block is dulpicated, same with `v1::RepoTeam`. We could use the From trait

Reply:
There's not much benefit to `From` here, because in the tests, it's more ergonomical to just use the atomic attributes (llike name and permissions), and it's not very obvious to implement e.g. `impl From<(&str, RepoPermission)> for v1::RepoMember {}`, nor would I want to do that :D